opennms-base-assembly/src/main/filtered/etc/collectd-configuration.xml

```diff
@@ -3,37 +3,29 @@
 <collectd-configuration
         threads="50">
 
-    <package name="cassandra21x">
-        <filter><![CDATA[(IPADDR != '0.0.0.0') & (categoryName == 'Cassandra21x')]]></filter>
+    <package name="cassandra-via-jmx">
+        <filter>IPADDR != '0.0.0.0'</filter>
         <service name="JMX-Cassandra" interval="300000" user-defined="false" status="on">
             <parameter key="port" value="7199"/>
             <parameter key="retry" value="2"/>
             <parameter key="timeout" value="3000"/>
             <parameter key="protocol" value="rmi"/>
             <parameter key="urlPath" value="/jmxrmi"/>
-            <parameter key="rrd-base-name" value="cassandra21x"/>
-            <parameter key="ds-name" value="cassandra21x"/>
-            <parameter key="friendly-name" value="cassandra21x"/>
-            <parameter key="collection" value="cassandra21x"/>
+            <parameter key="collection" value="cassandra30x"/>
+            <parameter key="friendly-name" value="cassandra"/>
```

Comment:
Just a question, is there anything we have to take care of, in case we move from cassandra 21x to cassandra3x on a system using a similar friendly name?